duckduckgo/chrome-zeroclickinfo
# Add Grunt and Handlebars

**Reviewer:** @jdorweiler 



## Description:

The existing extension has little infrastructure: no build system, no Handlebars templates, no Sass.
So, as a first step:
- Add Grunt and Handlebars to package.json
- Add Gruntfile with tasks to compile Handlebars templates and concatenate them with popup.js
- Add templates/ dir for Handlebars templates and static/js/ for the compiled result
- Take out hardcoded HTML inside popup.js and use two Handlebars templates instead, for tracking domains and request counts shown in the popup
- Import Handlebars and static JS in popup.html

The static JS is pushed as well for ease of testing

## Steps to test this PR:
Since we're now using templates for the trackers list, we want to test whether that function is intact and blocked domains are shown correctly.

1. Load any website
2. Open popup and look at the trackers list
3. Verify the list of trackers is shown correctly: domains and requests match the number of blocked trackers in the icon badge
4. Repeat from step 1. with other websites


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
